app/views/statistics/index.html.erb

```diff
@@ -1,6 +1,12 @@
 <% title _("Statistics"), _('Statistics') %>
-<div id="statistics">
-<% charts.each do |ch| %>
-  <%= content_tag(:div, ch, :class=>"stats-well") %>
-<% end %>
-</div>
+
+<div id="statistics"></div>
+
+<%= javascript_tag defer: 'defer' do -%>
```

Comment:
while it looks extra complicated - something along the lines of https://github.com/reactjs/react-rails/blob/master/lib/react/rails/component_mount.rb#L22 ?